history/characters/60000_63000_lordaeronian.txt

```diff
@@ -331,22 +331,23 @@
 	dynasty=30002
 	culture=lordaeronian religion=holy_light
 	martial=4 diplomacy=4 stewardship=4 intrigue=8 learning=8
-	trait=education_learning_4 trait=chaste trait=patient trait=sadistic
+	trait=education_learning_4 trait=lustful trait=deceitful trait=sadistic trait=arrogant
```

Comment:
> Pretty much everything about the way Jandice dresses, speaks, and behaves just screams Lustful to me.

I wouldn't judge by the dress because it's fantasy with armored bras. So except for the dress, I don't get why she's lustful.

Instead of Lustful I would go with Diligent to show that she's passionate enough to become an archmage.

Reply:
> I wouldn't judge by the dress because it's fantasy with armored bras. So except for the dress, I don't get why she's lustful.

In that case, I would urge you to listen to [her voice lines from WoW](https://www.youtube.com/watch?v=NlWWRMZ6CL4). There's a certain flirtatious kind of enunciation in the way she delivers some of her lines. But if this doesn't change your mind, I would be okay with Diligent as well.

Reply:
If those lines are not an attempt for a medieval character to be "flirty" playing with my emotions, then i do not know what is.